content/en/docs/instrumentation/java/getting-started.md

```diff
@@ -227,9 +238,12 @@ For more:
 - Try [automatic instrumentation](../automatic/) on one of your own apps.
 - For light-weight customized telemetry, try [annotations][].
 - Learn about [manual instrumentation][] and try out more
-  [examples]({{% relref examples %}}).
+[examples](../examples/).
```

Comment:
Oh, to get the link checker to be happy again, either refert this change to its use of `relref` or put an absolute path.

Reply:
thx, fixed :)